inference-engine/src/legacy_api/src/ngraph_ops/eltwise.cpp

```diff
@@ -9,6 +9,7 @@
 
 #include "ngraph/util.hpp"
 #include "ngraph/validation_util.hpp"
+#include "../itt.hpp"
```

Comment:
why do we need `..`? I don't see `itt.hpp` inside `legacy_api` folder..

Reply:
It is not ready. For some reasons `"itt.hpp` doesn't work. I didn't investigate it but of course I will fix it before the merge.